2_lesson/lesson_two.html

```diff
@@ -0,0 +1,177 @@
+
+// задание четвертое
+/*
+
+    let a = 8;
+
+    switch (a) {
+        case 0:
+            console.log(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 1:
+            console.log(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 2:
+            console.log(2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 3:
+            console.log(3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 4:
+            console.log(4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 5:
+            console.log(5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 6:
+            console.log(6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 7:
+            console.log(7, 8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 8:
+            console.log(8, 9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 9:
+            console.log(9, 10, 11, 12, 13, 14, 15);
+            break;
+        case 10:
+            console.log(10, 11, 12, 13, 14, 15);
+            break;
+        case 11:
+            console.log(11, 12, 13, 14, 15);
+            break;
+        case 12:
+            console.log(12, 13, 14, 15);
+            break;
+        case 13:
+            console.log(13, 14, 15);
+            break;
+        case 14:
+            console.log(14, 15);
+            break;
+        case 15:
+            console.log(15);
+            break;
+            defaul:
+            console.log('ERROR');
+    }
+
+*/
+
+//----------------------------------------------------//
+
+// задание пятое
+
+function addition (x, y) {
+    return x + y;
+}
+function subtraction (x, y) {
+    return x - y;
+}
+function multiplication (x, y) {
+    return x * y;
+}
+function division (x, y) {
+    return x / y;
+}
+
+//-------------------------------------------------------//
+
+// задание шестое
+/*
+
+function mathOperation(arg1, arg2, operation) {
+    switch (operation) {
+        case addition:
+        console.log(addition (arg1, arg2));
+        break;
+        case subtraction:
+        console.log(subtraction (arg1, arg2));
+        break;
+        case multiplication:
+        console.log(multiplication (arg1, arg2));
+        break;
+        case division:
+        console.log(division (arg1, arg2));
+        break;
+        default:
+        console.log('ERROR');
+    }
+}
+
+mathOperation(5, 2, division);
+
+*/
+
+//-----------------------------------------------------//
+
+// задание седьмое
+/*
+    console.log(null == 0);
+    при сравнении null и '0' получаем false. Так как сравниваем два разных типа данных
+*/
+
+//----------------------------------------------------//
+
+// задание восьмое
+/*
+
+function power(val, pow) {
+    if (pow == 1) {
+        return val;
+    } else {
+       return val * power(val, pow -1 );
```

Comment:
Будь немного аккуратнее в оформлении :) Читаемость кода в особенно больших командах - очень важно
```suggestion
       return val * power(val, pow - 1);
```